pkg/restapi/internal/common/support/httphandler.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package support
+
+import (
+	"net/http"
+
+	"github.com/trustbloc/sidetree-core-go/pkg/restapi/common"
+)
+
+// Handler http handler for each controller API endpoint.
+type Handler interface {
```

Comment:
probably no need for this common handler

Reply:
done